usr/share/rear/layout/save/GNU/Linux/510_current_disk_usage.sh

```diff
@@ -2,4 +2,4 @@
 # Public License. Refer to the included COPYING for full text of license.
 
 # Save the current disk usage (POSIX output format) in the rescue image
-df -Plh |grep -vE '^(encfs)' > $VAR_DIR/layout/config/df.txt
+df -Plh -x encfs -x tmpfs -x devtmpfs > $VAR_DIR/layout/config/df.txt
```

Comment:
@gdha To make sure that it stays backward compatible is why I included the FIXME message. My guess is that GNU `df` versions have the `x` option on board, and BSD variants do not.